units/host_termination.go

```diff
@@ -34,8 +34,9 @@ func init() {
 }
 
 type hostTerminationJob struct {
-	HostID   string `bson:"host_id" json:"host_id" yaml:"host_id"`
-	job.Base `bson:"metadata" json:"metadata" yaml:"metadata"`
+	HostID          string `bson:"host_id" json:"host_id"`
+	TerminateIfBusy bool   `bson:"terminate_if_busy" json:"terminate_if_busy"`
```

Comment:
we should decide (talk off line?) if we should bump the version of the document to prevent old code from running new jobs and vice versa.

Reply:
I don't think so - both scenarios will behave as if this is set to false (old code behavior). It'll just be as if this were deployed slightly later than it really were. Also tasks that go purple like this are restarted automatically